code-challenges.test.js

```diff
@@ -10,50 +10,122 @@
+    "chopsticks",
+    "mango",
+    "deduction",
+  ];
+  // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+  it("will use .map() and logic on all values in the array", () => {
+    // adding variables into the parameter since they are declared up top
+    expect(capitalize(randomNouns1)).toEqual([
+      "Streetlamp",
+      "Potato",
+      "Teeth",
+      "Conclusion",
+      "Nephew",
+    ]);
+    expect(capitalize(randomNouns2)).toEqual([
+      "Temperature",
+      "Database",
+      "Chopsticks",
+      "Mango",
+      "Deduction",
+    ]);
+  });
+});
 
 // b) Create the function that makes the test pass.
 
+// create a function that takes an array as argument √
+// use .map() to iterate over array and √
+// use charAt(0), .toUpperCase(), and .slice() in the rest of the word to make each value capitalized √
+// return array with a values capitalized √
+// at first I thought I only needed .toUpperCase() but that would have turned the entire word into all caps
 
+const capitalize = (array) => {
+  return array.map((value) => value.charAt(0).toUpperCase() + value.slice(1));
+};
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
-
 // a) Create a test with expect statements for each of the variables provided.
-
-var vowelTester1 = "learn"
-// Expected output: 1
-var vowelTester2 = "academy"
-// Expected output: 0
-var vowelTester3 = "challenges"
-// Expected output: 2
-
-
+describe("index of first vowel", () => {
+  var vowelTester1 = "learn";
+  // Expected output: 1
+  var vowelTester2 = "academy";
+  // Expected output: 0
+  var vowelTester3 = "challenges";
+  // Expected output: 2
+  it("will take in a string and log the index of the first vowel", () => {
+    expect(firstVowel(vowelTester1)).toEqual(1);
+    expect(firstVowel(vowelTester2)).toEqual(0);
+    expect(firstVowel(vowelTester3)).toEqual(2);
+  });
+});
 
 // b) Create the function that makes the test pass.
+
+// create a function that takes in a string as the argument √
+// create a list of vowels √
+// turn string into an array with split and use forEach √ (kinda, ended up using for loop)
+// use includes() to check if character is in vowels list, if so return index √
+// return the index √
+
+// const firstVowel = (string) => {
+//   let vowels = ["a", "e", "i", "o", "u", "y"];
+//   let splitString = string.split("");
+//   for (let i = 0; i < splitString.length; i++) {
+//     if (vowels.includes(splitString[i])) {
+//       return i;
+//     }
+//   }
+// };
+
+// I tried refactoring but couldn't get it to really work that well, here's what I ended up with by using .forEach() instead of a for loop but not really sastified.
+
+const firstVowel = (string) => {
+  let vowels = ["a", "e", "i", "o", "u", "y"];
+  let vowelIndexes = [];
+  string
+    .split("")
+    .forEach((val, i) => (vowels.includes(val) ? vowelIndexes.push(i) : null));
+  return vowelIndexes[0];
+};
```

Comment:
This works. This particular problem is interesting because you really only care about the first instance of the first vowel. Something to think about.